config/state.ts

```diff
@@ -46,6 +38,15 @@ type State = {
 	lastEvent?: Event;
 	lastEventTime?: number;
 	enableSFXConfetti?: boolean;
+	enableSound?: boolean;
```

Comment:
Please update this state key to `enableSFXSound` to follow the naming convention already set.